domains/certificates/Query.API/API/DataSyncSyncer/Startup.cs

```diff
@@ -1,9 +1,23 @@
+using System;
+using API.DataSyncSyncer.Client;
+using API.DataSyncSyncer.Configurations;
+using API.DataSyncSyncer.Persistence;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace API.DataSyncSyncer;
 
 public static class Startup
 {
-    public static void AddDataSyncSyncer(this IServiceCollection services)
-        => services.AddHostedService<DataSyncSyncerWorker>();
+    public static void AddDataSyncSyncer(this IServiceCollection services, IConfiguration configuration)
+    {
+        var datasyncUrl = configuration.GetSection(DatasyncOptions.Datasync).Value;
+        services.AddHttpClient<DataSyncService>(client => client.BaseAddress = new Uri(datasyncUrl));
```

Comment:
This does not correctly bind an instance of DatasyncOptions. This works:

```suggestion
        DatasyncOptions options = new();
        configuration.GetRequiredSection(DatasyncOptions.Datasync).Bind(options);
        services.AddHttpClient<DataSyncService>(client => client.BaseAddress = new Uri(options.Url));
```